### PR TITLE
Fix linreg docs

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

File: python/hail/methods/statgen.py [hunk 1]
```diff
@@ -13,41 +13,51 @@
...
+    ys : :obj:`list` of :class:`hail.expr.expression.Expression`
+        One or more response expressions.
+    x : :class:`hail.expr.expression.Expression`
+        Input variable.
+    covariates : :obj:`list` of :class:`hail.expr.expression.Expression`
+        Covariate expressions.
+    root : :obj:`str`
+        Name of resulting row-indexed field.
+    block_size : :obj:`int`
+        Number of row regressions to perform simultaneously. Larger blocks
```
Comment (jbloom22) [inline python/hail/methods/statgen.py]: I think it's clearer to say "simultaneously per core".

File: python/hail/methods/statgen.py [hunk 2]
```diff
@@ -13,41 +13,51 @@
...
+    ...                                 covariates=[dataset.pheno.age, dataset.pheno.isFemale])
+
+    Warning
+    -------
+    :meth:`linreg` uses the same set of columns for each phenotype, the set of
+    columns for which **all** response variables and covariates are defined.
+
+    Notes
+    -----
+
```
Comment (jbloom22) [inline python/hail/methods/statgen.py]: We lost the model so it's very confusing what is actually being returned (e.g. what is `\hat\beta_1`?). How about I make a PR adding the context back in once this PR is merged?

Comment (tpoterba) [reply python/hail/methods/statgen.py]: yeah, sounds good.

File: python/hail/methods/statgen.py (hunk 1 above)
Comment (tpoterba) [reply python/hail/methods/statgen.py]: make that fix in your PR, not worth retesting!

**Pull request merged**